tests/sles4sap/qesapdeployment/deploy.pm

```diff
@@ -27,6 +27,7 @@ sub run {
         my $rec_timeout = qesap_ansible_log_find_timeout($ret[1]);
         if ($rec_timeout) {
             record_info('DETECTED ANSIBLE TIMEOUT ERROR');
+            @ret = qesap_execute(cmd => 'ansible', cmd_options => '--profile', verbose => 1, timeout => 3600);
```

Comment:
I think it is not ok as in case of Ansible failure and then Ansible pass at the retry, the `die` is called in any case

Reply:
ok solved as the behavior is intentional. We do not know yet what is going to happens to run Ansible for a second time in a row. So it is fine to do it but to `die` in any case.